tests/performance_tests/performance_test_basic.py

```diff
@@ -566,11 +606,12 @@ def main():
     netPluginArgs = NetPluginArgs(netThreads=args.net_threads)
     ENA = PerformanceTestBasic.ClusterConfig.ExtraNodeosArgs
     extraNodeosArgs = ENA(chainPluginArgs=chainPluginArgs, httpPluginArgs=httpPluginArgs, producerPluginArgs=producerPluginArgs, netPluginArgs=netPluginArgs)
+    SC = PerformanceTestBasic.ClusterConfig.SpecifiedContract
+    specifiedContract=SC(accountName=args.account_name, ownerPublicKey=args.owner_public_key, activePublicKey=args.active_public_key,
```

Comment:
Why is it we don't use the constructor directly?

Reply:
I factored the construction of the `SpecifiedContract` object out to help make the `PtbConfig` constructor a little easier to read and understand instead of cramming all of the arguments to each sub object into the same constructor input.  Same as was done for `ExtraNodeosArgs`.  Just trying to keep each piece a little easier to read and comprehend.